### PR TITLE
Login signup ui part done rn..  addons will be done later

### PR DESCRIPTION
**Pull request by aswathcm29**

### CODE REVIEW

File: FrontEnd/src/components/LoginCred/Login.helper.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        onChange={(e) => setPassword(e.target.value)}
+        required
+        className='block w-72 py-2.3 px-0 text-sm text-white bg-transparent border-0 border-b-2 borer-gray-300 appearance-none dark:focus:border-blue-500 focus:outline-none focus:ring-0 focus-text-white focus:border-blue-600 peer '
+      />
+      <label className='absolute text-sm text-white duration-300 transform -translate-y-6 scale-75 top-1 -z-10 origin-[0] peer=focus:left-0 peer-focus:text-blue-600 peer-focus:dark:text-blue-500 peer-placeholder-shown:scale-100 peer-placeholder-shown:translate-y-0 peer-focus:scale-75 peer-focus-translate-y-6 '>Password</label>
+      </div>
+      <br />
+      <p>Don't have an account ? <button className='' onClick={() => toSignup()}>New here?</button></p>
+      <label>{errorMessage}</label>
+      <button type="submit">Login</button>
```
Comment (muruga21) [inline FrontEnd/src/components/LoginCred/Login.helper.js:77]: add some style to this button

File: FrontEnd/src/components/LoginCred/Login.helper.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                        onChange={(e)=>setPassword(e.target.value)}
+                        required
+                        className='block w-72 py-2.3 px-0 text-sm text-white bg-transparent border-0 border-b-2 border-gray-300 appearance-none dark:focus:border-blue-500 focus:outline-none focus:ring-0 focus-text-white focus:border-blue-600 peer-text-6xl'
+                    />
+                   <label htmlFor='' className='absolute text-sm text-white duration-300 transform -translate-y-6 scale-75 top-1 -z-10 origin-[0] peer-focus:left-0 peer-focus:text-blue-600 peer-focus:dark:text-blue-500 peer-placeholder-shown:scale-100 peer-placeholder-shown:translate-y-0 peer-focus:scale-75 peer-focus-translate-y-6 '>Password</label>
+                <br/>
+                </div>
+                <p>Already have an account ? <button className='' onClick={()=>toLogin()}>Login</button></p>
+                <label>{errorMessage}</label>
+                <button type="submit">Sign Up</button>
```
Comment (muruga21) [inline FrontEnd/src/components/LoginCred/Login.helper.js:168]: style needed

File: FrontEnd/src/components/LoginCred/Login.helper.js (hunk 1 above)
Comment (aswathcm29) [reply FrontEnd/src/components/LoginCred/Login.helper.js]: Added now ! Please do review my recent commit for the updated changes

**Review by Cibiyanna26 — APPROVED** [review]

**Comment by Cibiyanna26** [comment]
going to merge to dashboard

**Pull request merged**